addon/components/chart-component.js

```diff
@@ -48,7 +48,7 @@ export default Ember.Component.extend( ColorableMixin, ResizeHandlerMixin, {
 
   // Transform the view commonly displaced by the margin
   transformViewport: Ember.computed('marginLeft', 'marginTop', function() {
-    return ('translate(' + this.get('marginLeft') + ',' + this.get('marginTop') + ')');
+    return 'translate(' + this.get('marginLeft') + ',' + this.get('marginTop') + ')';
```

Comment:
I thought about doing stuff like that, but at the moment I'd like to avoid doing additional refactoring, as my goal is to get this merged tonight. We can go over the codebase again looking for things like this later.